ruleset/rules/0755-office365_rules.xml

```diff
@@ -1140,39 +1141,39 @@ ID: 91531 - 91650
         <field name="office365.Operation" type="osregex">^QuarantineDelete$</field>
         <description>Office 365: Deleted quarantine message</description>
         <options>no_full_log</options>
-        <group>Exchange</group>
+        <group>Exchange,hipaa_164.312.a.1,hipaa_164.312.b,pci_dss_5.3,pci_dss_10.6.1,pci_dss_11.4,pci_dss_11.5</group>
     </rule>
 
     <rule id="91720" level="12">
         <if_sid>91532</if_sid>
         <field name="office365.Operation" type="osregex">^QuarantineExport$</field>
         <description>Office 365: Exported quarantine message</description>
         <options>no_full_log</options>
-        <group>Exchange</group>
+        <group>Exchange,hipaa_164.312.a.1,hipaa_164.312.b,pci_dss_5.3,pci_dss_10.6.1,pci_dss_11.4,pci_dss_11.5</group>
     </rule>
 
     <rule id="91721" level="6">
         <if_sid>91532</if_sid>
         <field name="office365.Operation" type="osregex">^QuarantinePreview$</field>
         <description>Office 365: Previewed quarantine message</description>
         <options>no_full_log</options>
-        <group>Exchange</group>
+        <group>Exchange,hipaa_164.312.a.1,hipaa_164.312.b,pci_dss_5.3,pci_dss_10.6.1,pci_dss_11.4,pci_dss_11.5</group>
     </rule>
 
     <rule id="91722" level="12">
         <if_sid>91532</if_sid>
         <field name="office365.Operation" type="osregex">^QuarantineRelease$</field>
         <description>Office 365: Released quarantine message</description>
-        <options>no_full_log</options>
-        <group>Exchange</group>
+        <options>no_full_log,pci_dss_10.2.2</options>
```

Comment:
The compliance tag is on the options and it's making the checks fail

Reply:
solved